ddb_cp.module

```diff
@@ -243,19 +255,21 @@ function ddb_cp_ajax_test_result($site = NULL) {
   else {
     return MENU_NOT_FOUND;
   }
+
+  // @todo: Missing return statement it will note return any thing if the if statement is hit?
```

Comment:
Please try the control panel and you'll see that it actually works!

drupal_json_output() has an implicit return and finishes the HTTP request.